main.tf.json

```diff
@@ -1140,6 +1140,7 @@
                         "time_sleep.wait_before_create"
                     ],
                     "name": "ClumioEC2ProtectManagedPolicy-${var.aws_region}-${var.clumio_token}",
+                    "path": "/clumio/",
```

Comment:
Is there a reason why `clumio_discover_policy` is excluded from having a path too?

Reply:
Added path to clumio_discover_policy.